public/static/docs/command-reference/import.md

```diff
@@ -160,23 +159,14 @@ deps:
 If `rev` is a Git branch or tag (where the underlying commit changes), the data
 source may have updates at a later time. To bring it up to date if so (and
 update `rev_lock` in the DVC-file), simply use `dvc update <stage>.dvc`. If
-`rev` is a specific commit hash (does not change), `dvc update` will never have
-an effect on the import stage. You may **re-import** a different commit instead,
-by using `dvc import` again with a different (or without) `--rev`. For example:
+`rev` is a specific commit (does not change), `dvc update` will never have an
+effect on the import stage. You may `dvc update` to a different commit, using
+`--rev`:
 
 ```dvc
-$ dvc import --rev master \
-             git@github.com:iterative/dataset-registry.git \
-             use-cases/cats-dogs
+$ dvc update --rev cats-dogs-v2
 ```
 
-The import stage is overwritten, and will now be able update normally with
-`dvc update`.
-
-> In the above example, the value for `rev` in the new import stage will be
-> `master` (default branch), so the command is equivalent to not using `--rev`
-> at all.
```

Comment:
This info is still useful if using --rev master above, just a little different:

```markdown
> In the above example, the value for `rev` in the new import stage will be
> `master` (a branch) so it will be able update normally going forward.
```